src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -304,6 +310,7 @@ class ComposeUp extends DefaultTask {
         def containerId = getContainerId(serviceName)
         new ByteArrayOutputStream().withStream { os ->
             project.exec { ExecSpec e ->
+                extension.setExecSpecWorkingDirectory(e)

```

Comment:
We execute just `docker` command that is current directory agnostic IMHO. So we could remove this line.